influxdb/client.py

```diff
@@ -252,7 +252,8 @@ def request(self, url, method='GET', params=None, data=None,
         else:
             raise InfluxDBClientError(response.content, response.status_code)
 
-    def write(self, data, params=None, expected_response_code=204):
+    def write(self, data, params=None, expected_response_code=204,
+              protocol='json'):
```

Comment:
Should update doc string for `data` type (dict for json protocol, sequence of lines for line protocol.)